### ISSUE TITLE
GetSetElements - Incorrect port range/interval parsing (or not supported at all)

### ISSUE THREAD

**Issue by msdean** [google/nftables#154]
I'm using latest nftables:
nftables (bcad476126f07691b7bc3657d1ade257d469e715)
libmnl (2a3e927ec79e3a52d8cea60ccdfb8699b8a0d56e)
libnftnl (e2514c0eff4da7e8e0aabd410f7b7d0b7564c880)
On kernel 5.13 (Ubuntu 20.04LTS).

I have the following concatenated set:
```
set ip6_set{
                type ipv6_addr . inet_service . ipv6_addr
                flags interval
                elements = {
                             777c:ab4b:85f0:1614:49e5:d29b:aa7b:cc90 . 1-60000 . 8709:1cb9:163e:9b55:357f:ef64:708a:edcb }
        }
```

Emphasis on the port range 1-60000.
When I'm printing the set and its elements I get the following:
The set itself:
```
&nftables.Set{Table:(*nftables.Table)(0xc0000a0160), ID:0x0, Name:"ip6_set", Anonymous:false, Constant:false, Interval:true, IsMap:false, HasTimeout:false, Timeout:0, KeyType:nftables.SetDatatype{Name:"", Bytes:0x0, nftMagic:0x8348}, DataType:nftables.SetDatatype{Name:"", Bytes:0x0, nftMagic:0x0}}
```

The set element:
```
nftables.SetElement{Key:[]uint8{0x77, 0x7c, 0xab, 0x4b, 0x85, 0xf0, 0x16, 0x14, 0x49, 0xe5, 0xd2, 0x9b, 0xaa, 0x7b, 0xcc, 0x90, 0x0, 0x1, 0x0, 0x0, 0x87, 0x9, 0x1c, 0xb9, 0x16, 0x3e, 0x9b, 0x55, 0x35, 0x7f, 0xef, 0x64, 0x70, 0x8a, 0xed, 0xcb}, Val:[]uint8(nil), IntervalEnd:false, VerdictData:(*expr.Verdict)(nil), Timeout:0}
```

Specifically, note the middle part that corresponds to the port range (after the `0x90` and before the `0x87`):
```
nftables.SetElement{Key:[]uint8{...0x90, 0x0, 0x1, 0x0, 0x0, 0x87,...}
```

As you can see, the first value is parsed correctly (0x0, 0x1 == 1), but the other part isn't (0x0, 0x0 == 0).

To confirm, when I execute `SetAddElements` with the above element, I get back:
```
777c:ab4b:85f0:1614:49e5:d29b:aa7b:cc90 . 1 . 8709:1cb9:163e:9b55:357f:ef64:708a:edcb
```
Which is of course is incorrect.

Trying to change the zeroed values doesn't seem to change anything.
Am I doing something wrong or is this simply not supported? 
The set definition does specify `Interval:true` so it does seem to be supported.

EDIT:
Ok, I realized that adding 2 identical entries, one with start port, the other with end port and `IntervalEnd:true` actually ends up with a single entry and a port range. But I don't see 2 elements when I'm trying to GET the elements if they were added with the nft cmdline tool. Why is that?

To summarize:
When using nft cmdline tool, a set entry with a port range is created, but when trying to access it via go code, I only see a single entry with the port range "from" value (without the "to" value).
Using go code to create 2 entries, one with the "from" port, the other with the "to" port (and `IntervalEnd:true`) results in 2 entries when looking at it from the go code, and a single port-range entry when looking at it with the nft cmdline too.

**Comment by stapelberg** [comment]
I think it’d be best to write a test which compares the output of the `nft` command line tool and what this nftables package produces, and then debug from there. We’ll need a test anyway for submitting a fix, and perhaps the problem is obvious from the byte differences.

**Comment by msdean** [comment]
This is great! Much appreciated!

**Comment by msdean** [comment]
This is great! Much appreciated!
Thank you @turekt!
